src/sentry/integrations/jira/webhooks.py

```diff
@@ -62,8 +62,24 @@ def post(self, request, *args, **kwargs):
         except (KeyError, IndexError):
             return self.respond(status=400)
 
+        try:
+            integration = get_integration_from_jwt(
+                token, request.path, 'jira', request.GET, method='POST'
+            )
+        except AtlassianConnectValidationError:
+            return self.respond(status=400)
+
         data = request.DATA
 
+        if not data.get('changelog'):
+            logger.info(
+                'missing-changelog', extra={
+                    'integration_id': integration.id,
+                    'data': data,
+                }
+            )
+            return self.respond()
```

Comment:
Is there a reason we want to return a success instead of a 400?

Reply:
it's kind of hard to tell what to expect of this webhook based on jira's docs (https://developer.atlassian.com/cloud/jira/software/webhooks/) but it seemed plausible that there are maybe some cases where the changelog wouldn't be included, so i wasn't completely sure it should be considered a bad request if it's missing.